packages/types/src/index.ts

```diff
@@ -37,8 +35,10 @@ export interface RequestArguments {
 }
 
 // per EIP-1193
```

Comment:
nit: Anywhere it says "per EIP-X", could you update to actually include the link to the EIP, eg

   // @see https://eips.ethereum.org/EIPS/eip-1193

Reply:
Yeah, I can. Unfortunately, I can't link to specific headings, which makes it particularly hard to reference specific points, but overall, this is good improvement.

Reply:
Sorry, I can, will link to Github instead